spec/spec_helper.rb

```diff
@@ -27,6 +27,8 @@
 require 'event_store_client'
 require 'timecop'
 require 'dry-schema'
+require 'dry-monads'
+require 'dry/monads/result'
```

Comment:
why are these still needed?

Reply:
Explained above ^
We still need to test `#key_repository` is supposed to return `Dry::Monads::Result`